Nodejs/Product/ProjectWizard/ImportWizard.cs

```diff
@@ -29,6 +29,7 @@ public sealed class NewProjectFromExistingWizard : IWizard {
         public void ProjectFinishedGenerating(EnvDTE.Project project) { }
         public void ProjectItemFinishedGenerating(EnvDTE.ProjectItem projectItem) { }
         public void RunFinished() { }
+        public static Boolean AddingNewProject { get; set; } = false;
```

Comment:
> Make this internal

This cannot be internal, as it is referenced in ``NodejsPackage.Initialize()`` which is not in the same  namespace.